### DISCUSSION TITLE
Write FastHTML examples

### DISCUSSION THREAD

**Discussion by stephenhibbert** [awslabs/aws-lambda-web-adapter#495] [Ideas]
Add support for [FastHTML](https://fastht.ml/) - a modern Python + HTMX + ASGI + Uvicorn + Starlett based web framework.

Publish examples for container and zip packaging, and for streaming.

**Comment by bnusunny** [comment]
FastHTML is nice! Would you like to contribute these examples?

**Reply by stephenhibbert** [reply]
I've actually already started! Will get a pull request together :)

**Reply by stephenhibbert** [reply]
#496

**Comment by bnusunny** [comment]
Awesome! I left one comment on the PR. It is not a blocking issue.
